Bastion/src/isaac/bastion/manager/BastionBlockManager.java

```diff
@@ -287,145 +280,4 @@ public String infoMessage(boolean dev, Block block, Block clicked, Player player
 
 		return sb.toString();
 	}
-
-	public void handleBlockPlace(BlockPlaceEvent event) {
```

Comment:
More missing events here ... where is equivalent? And were these even doing anything since they lack annotations?